src/framework/constraints.ml

```diff
@@ -15,10 +15,10 @@ sig
 end
 
 
-(** The main point of this file---generating a [GlobConstrSys] from a [Spec]. *)
+(** The main point of this file---generating a [DemandGlobConstrSys] from a [Spec]. *)
 module FromSpec (S:Spec) (Cfg:CfgBackward) (I: Increment)
   : sig
-    include GlobConstrSys with module LVar = VarF (S.C)
+    include DemandGlobConstrSys with module LVar = VarF (S.C)
                            and module GVar = GVarF (S.V)
                            and module D = S.D
                            and module G = GVarG (S.G) (S.C)
```

Comment:
Maybe we want the `and` to still align with `with`. Alternatively, we should go for  a more standard indentation.

Reply:
For some reason, `ocp-indent` wants to align it with the `i` of `with`. (maybe two spaces after the end of the name?) Went with that since I cannot commit anything else anyway.

Reply:
It was also aligned like this before. I think it aligns the `module`-s.